spec/simple_segment/client_spec.rb

```diff
@@ -27,7 +36,11 @@
         'anonymousId' => nil,
         'traits' => {
           'name' => 'Philip J. Fry',
-          'occupation' => 'Delivery Boy'
+          'occupation' => 'Delivery Boy',
+          'foo_time' => foo_time.utc.iso8601(3),
+          'foo_utc_time' => foo_utc_time.iso8601(3),
+          'foo_date_time' => foo_date_time.to_time.utc.iso8601(3),
+          'foo_date' => foo_date.to_time.utc.iso8601(3)
```

Comment:
Yeah, that makes sense, here it is: https://github.com/whatthewhat/simple_segment/pull/19/commits/7a3cc403c287d31986ff61c23013be5c56a2037e.
Please note that in order to keep the original time zone in `maybe_datetime_in_iso8601` I have used the `let expected value` in the assertion for `date_time` [here](https://github.com/whatthewhat/simple_segment/pull/19/commits/7a3cc403c287d31986ff61c23013be5c56a2037e#diff-ef2d4e1f36661d47cd539db8657fb3bfR39).
I can't use a more specific value for `datetime` because when using `to_time` [here](https://github.com/whatthewhat/simple_segment/pull/19/commits/7a3cc403c287d31986ff61c23013be5c56a2037e#diff-34280fe876f1742cec2ff85e72831f96R33), the result depends on the zone.
If you have any ideas for this, I'll be happy to include them

Reply:
No problem, let's leave it the way you have it now 👍